### PR TITLE
fix: updateMany use undefined variable;

### PR DESCRIPTION
**Pull request by DenTray**
feat: implement is_multidimensional helper, update docs

refs: #33

### CODE REVIEW

File: documentation/traits.md [hunk 1]
```diff
@@ -9,13 +9,27 @@ Provides CRUD-based methods to work with database-related entities.
...
+- `assertEqualsFixture($fixture, $data, bool $exportMode = false)`
+- `exportJson($fixture, $data)`
+- `exportContent($content, $fixture)`
+- `exportFile($response, $fixture)`
+
+### DB prefilling
+
+ Trait also provides an ability to use the same initial database state before
+each test running. Testing database will be automatically cleared in the `setUp`
+method and restore from the `/tests/fixtures/{$testClassName}/dump.sql` file.
```
Comment (astorozhevsky) [inline documentation/traits.md:32]:
```suggestion
method and restored from the `/tests/fixtures/{$testClassName}/dump.sql` file.
```

File: documentation/traits.md [hunk 2]
```diff
@@ -9,13 +9,27 @@ Provides CRUD-based methods to work with database-related entities.
...
 ## FixturesTrait
 
-This trait is designed to make testing understandable and cleaner.
-All auxiliary data such as the results of the operation can be placed in a way corresponding 
-to the following mask **base_path("tests/fixtures/{$testClassName}/{$fixture}")**, 
-and is easily obtained by the method *$this->getFixture($fixture)* or through *$this->getJsonFixture($fixture)*, 
-in which will be performed automatically decode json-data.
-Also you can tune your TestCase for restore dump of database witch will be places in 
-**base_path("tests/fixtures/{$testClassName}/dump.sql")** and method for comfortable getting of Json-responses
+Is a powerful tool which make testing process understandable and cleaner.
```
Comment (astorozhevsky) [inline documentation/traits.md:12]:
```suggestion
This is a powerful tool which makes testing process understandable and cleaner.
```

File: documentation/traits.md [hunk 3]
```diff
@@ -9,13 +9,27 @@ Provides CRUD-based methods to work with database-related entities.
...
-**base_path("tests/fixtures/{$testClassName}/dump.sql")** and method for comfortable getting of Json-responses
+Is a powerful tool which make testing process understandable and cleaner. 
+
+### Fixtures
+
+All additional data such as the results of the operation or input data can be
+presented via `.json` files and grouped by the test case name in the path
+`/tests/fixtures/{$testClassName}`.
+
+The most common fixtures helpers presented below:
```
Comment (astorozhevsky) [inline documentation/traits.md:20]:
```suggestion
The main fixture helpers are following:
```

**Review by astorozhevsky — CHANGES_REQUESTED** [review]

**Review by astorozhevsky — APPROVED** [review]

**Comment by DenTray** [comment]
Refs: #33

**Pull request merged**